yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher.java

```diff
@@ -326,16 +333,21 @@ public DockerComputerLauncher getPreparedLauncher(String cloudId, DockerSlaveTem
         cloneJNLPlauncher.setUser(getUser());
         cloneJNLPlauncher.setJvmOpts(getJvmOpts());
         cloneJNLPlauncher.setSlaveOpts(getSlaveOpts());
+        cloneJNLPlauncher.setTunnel(getTunnel());
         cloneJNLPlauncher.setJenkinsUrl(getJenkinsUrl());
         cloneJNLPlauncher.setNoCertificateCheck(isNoCertificateCheck());
         cloneJNLPlauncher.setNoReconnect(isNoReconnect());
 
         return cloneJNLPlauncher;
     }
 
+    public String getTunnel() {
```

Comment:
Please move setter after coressponding setter. Class organised in the next way: firstly field getters/setters, then overrides and additional methods